andela/blackninjas-frontend
# #69817828 trip request search

#### What does this PR do?
this PR comes to add search bar that will help the user and manager to search trip request
#### Description of Task to be completed?
	Search Function

 **Why**
As requests and approval tables grow, it should be easy to find requests through the search functionality

**As a user,** I should be able to use the search component
So that, I can easily retrieve records from both the request and approval table

 **Acceptance Criteria**
- Given that a user is logged in, when the user enters data into the search component, then the records displayed in the table should reflect all possible records relevant to the content of the  data entered by the user in the search component
- User should be able to search based on the column headers which are: request ID, owner, destination, origin, duration, start date, request status etc

#### How should this be manually tested?
After clone repository goes to the root of the project open path into command run the npm install and once setup is done type npm install.

after that type npm start you will get a sign-in page then sign in and you will be redirected to
the dashboard after successfully sign in
and then you will get trip requests to link on left side and manager will be able to see approvals table

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#69817828](https://www.pivotaltracker.com/story/show/169817828)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/47531860/77403076-55da4780-6db8-11ea-8391-8e4fff0e381e.png)

![image](https://user-images.githubusercontent.com/47531860/77544399-9d90ca00-6eb1-11ea-9d39-fe8117482daa.png)

#### Questions:
N/A
